tests/integration/hpe_3par_api_file_persona_test.py

```diff
@@ -0,0 +1,320 @@
+import pytest
+import time
+import docker
+import yaml
+import os
+from .base import TEST_API_VERSION, BUSYBOX
+from . import helpers
+from .helpers import requires_api_version
+from .hpe_3par_manager import HPE3ParBackendVerification,HPE3ParVolumePluginTest
+
+# Importing test data from YAML config file
+with open("testdata/test_config.yml", 'r') as ymlfile:
+    cfg = yaml.load(ymlfile)
+
+# Declaring Global variables and assigning the values from YAML config file
+
+PLUGIN_TYPE = cfg['plugin']['type']
+HOST_OS = cfg['platform']['os']
+THIN_SIZE = cfg['volumes']['thin_size']
+FULL_SIZE = cfg['volumes']['full_size']
+DEDUP_SIZE = cfg['volumes']['dedup_size']
+COMPRESS_SIZE = cfg['volumes']['compress_size']
+
+if PLUGIN_TYPE == 'managed':
+    HPE3PAR = cfg['plugin']['managed_plugin_latest']
+    CERTS_SOURCE = cfg['plugin']['certs_source']
+else:
+    HPE3PAR = cfg['plugin']['containerized_plugin']
+    PLUGIN_IMAGE = cfg['plugin']['containerized_image']
+    if HOST_OS == 'ubuntu':
+        PLUGIN_VOLUMES = cfg['ubuntu_volumes']
+    elif HOST_OS == 'suse':
+        PLUGIN_VOLUMES = cfg['suse_volumes']
+    else:
+        PLUGIN_VOLUMES = cfg['rhel_volumes']
+
+@requires_api_version('1.21')
+class FilePersonaTest(HPE3ParBackendVerification,HPE3ParVolumePluginTest):
+
+    @classmethod
+    def setUpClass(cls):
+        if PLUGIN_TYPE == 'managed':
+            c = docker.APIClient(
+                version=TEST_API_VERSION, timeout=600,
+                **docker.utils.kwargs_from_env()
+                )
+            try:
+                prv = c.plugin_privileges(HPE3PAR)
+                logs = [d for d in c.pull_plugin(HPE3PAR, prv)]
+                assert filter(lambda x: x['status'] == 'Download complete', logs)
+                if HOST_OS == 'ubuntu':
+                    c.configure_plugin(HPE3PAR, {
+                        'certs.source': CERTS_SOURCE
+                    })
+                else:
+                    c.configure_plugin(HPE3PAR, {
+                        'certs.source': CERTS_SOURCE,
+                        'glibc_libs.source': '/lib64'
+                    })
+                pl_data = c.inspect_plugin(HPE3PAR)
+                assert pl_data['Enabled'] is False
+                while pl_data['Enabled'] is False:
+                    c.enable_plugin(HPE3PAR)
+                    HPE3ParBackendVerification.hpe_wait_for_all_backends_to_initialize(cls, driver=HPE3PAR, help='backends')
+                pl_data = c.inspect_plugin(HPE3PAR)
+                assert pl_data['Enabled'] is True
+            except docker.errors.APIError:
```

Comment:
In setup class it is trying to install and get up plugin in running state
so in that case if we receive any error from docker server then it will except those